js/services/tvshows.js

```diff
@@ -0,0 +1,48 @@
+(function(annyang) {
+  'use strict';
+
+  function TVShowService($window, $http, $q) {
```

Comment:
It doesn't look like you need to pass in `$window` here. Can you either remove it or explain what it's there?